src/main/java/net/smartcosmos/dao/relationships/RelationshipDao.java

```diff
@@ -36,9 +54,19 @@
      */
     void delete(String accountUrn, String urn) throws IllegalArgumentException;
 
-    // TODO: add Look Up All Relationships Between Entities
+    /**
+     * Updates the moniker of a relationship
+     *
+     * @param accountUrn
+     * @param relationshipUpdateMoniker
+     * @return an {@link RelationshipResponse} instance for the created object
+     * @throws ConstraintViolationException if the {@link RelationshipCreate} violates constraints enforced by the persistence service
+     * @throws IllegalArgumentException if the relationship references by URN does not exist.*
+     */
+    RelationshipResponse updateMoniker(String accountUrn, RelationshipUpdateMoniker relationshipUpdateMoniker)
```

Comment:
That should be a "generic" `update()` method, not tied to just `moniker`. As I already mentioned, other implementations of the interface might allow other fields to be updated as well. If that actually makes sense is another question...